packages/experimental/Badge/src/Badge.types.ts

```diff
@@ -23,11 +23,21 @@ export interface BadgeConfigurableProps {
    */
   badgeColor?: BadgeColor;
 
+  /**
+   * The icon color.
+   */
+  iconColor?: ColorValue;
+
   /**
    * Badge position
    * @defaultvalue absolute
    */
   position?: FlexStyle['position'];
+
+  /**
+   * Text color.
+   */
+  textColor?: ColorValue;
```

Comment:
How is this prop different from color? Is there a reason to have both?

Reply:
I exposed textColor as a prop. When I tried to use color it didn't work. But I'll try again. Maybe I did smth wrong. It should work for color